index.bs

```diff
@@ -803,9 +774,30 @@ spec: mimesniff; urlPrefix: https://mimesniff.spec.whatwg.org/#
               value in {{MediaCapabilitiesEncodingInfo/configuration}}.
             </li>
             <li>
-              If the user agent is able to encode the media represented by
-              <var>configuration</var>, set {{MediaCapabilitiesInfo/supported}}
-              to <code>true</code>. Otherwise set it to <code>false</code>.
+              If <code>configuration.video</code> [=map/exists=], let
+              |videoMimeSupport| be the result of running [$Check MIME Type
+              Support$] with <code>configuration.video.\[[mimeType]]</code>,
+              the |media| video, and |configuration|'s
+              {{MediaEncodingConfiguration/type}}, otherwise
+              let |videoMimeSupport| be <code>undefined</code>.
```

Comment:
The `<code>configuration.video.\[[mimeType]]</code>`... that should link to what each part is. 

```suggestion
              |videoMimeSupport| be the result of running [=Check MIME Type
              Support=] with <code>configuration.video.\[[mimeType]]</code>,
              the |media| video, and |configuration|'s
              {{MediaEncodingConfiguration/type}}, otherwise
              let |videoMimeSupport| be `undefined`.
```